draft-ietf-quic-qlog-h3-events.md

```diff
@@ -71,7 +72,8 @@ of this document.
 This document defines a qlog event schema ({{Section 8 of QLOG-MAIN}})
 containing concrete events for the core HTTP/3 protocol {{RFC9114}} and selected
 extensions ({{!EXTENDED-CONNECT=RFC9220}}, {{!H3_PRIORITIZATION=RFC9218}}, and
-{{!H3-DATAGRAM=RFC9297}}).
+{{!H3-DATAGRAM=RFC9297}}), as well as the Capsule Protocol defined in
+{{!H3-DATAGRAM=RFC9297}}.
```

Comment:
We could probably make this part clearer with something like 

extensions: Extended CONNECT ({{!EXTENDED-CONNECT=RFC9220}}), PRIORITY_UPDATE ({{Section 7 of !PRIORITIZATION=RFC9218}}), and HTTP/3 Datagrams {{Section 2.1 of !DGRAM-AND-CAPSULE=RFC9297}}). It also defines a schema for the HTTP Capsule Protocol ({{Section 3 of DGRAM-AND-CAPSULE}}.